sdk_contrib/aws_v1_support/src/main/java/io/opentelemetry/sdk/contrib/trace/aws/Ec2Resource.java

```diff
@@ -46,19 +47,29 @@ public static Resource getResource() {
   // This can be tested now with a fake info and host.
   static Resource getResourceFromInfoAndHost(
       @Nullable InstanceInfo info, @Nullable String hostname) {
-    Map<String, String> labels = new HashMap<>();
-    labels.put(ResourceConstants.CLOUD_PROVIDER, CLOUD_PROVIDER_AWS);
+    Map<String, AttributeValue> labels = new HashMap<>();
+    labels.put(
+        ResourceConstants.CLOUD_PROVIDER, AttributeValue.stringAttributeValue(CLOUD_PROVIDER_AWS));
```

Comment:
Make CLOUD_PROVIDER_AWS an AttributeValue, also everywhere where we have constants for values.

Reply:
Beside in testing code, this is the only constant resource value we have! 
Done in 173c59d